### PR TITLE
HOTFIX: Frontend branch

### PR DESCRIPTION
**Pull request by toilavic**

### CODE REVIEW

File: frontend/src/components/MainContent/MapLeft/index.tsx [hunk 1]
```diff
@@ -25,65 +24,30 @@ const MapLeft: React.FC<Props> = ({ selectStore }) => {
...
-        if(a.name > b.name) { return 1 };
-        return 0
-      }) 
-
-      // tempStores = stores.sort(function(a:any,b:any) {
-      //   return a.{stores.forEach(getAvgRate(stores.rate))}
-      // }
-      // )
-      //console.log(tempStores);
+      var tam = stores.map((store:any) => {
```
Comment (vicnguyenqt) [inline frontend/src/components/MainContent/MapLeft/index.tsx:42]: Again, please change the variable name into something meaningful. This is not a competition that who win the best prize by naming shortless

File: frontend/src/components/MainContent/MapLeft/index.tsx [hunk 2]
```diff
@@ -25,65 +24,30 @@ const MapLeft: React.FC<Props> = ({ selectStore }) => {
...
+        store.rateArv = getAvgRate(store.rate);
+        return store;
+      });
+      tempStores = tam.sort(function(a:any, b:any) {
```
Comment (vicnguyenqt) [inline frontend/src/components/MainContent/MapLeft/index.tsx:46]: Variable naming, what is "a" and "b"? Meaning something?

**Comment by vicnguyenqt** [comment]
Looks good! A few remarks:

Consistency
- const/let or var?
- Lines ending to semicolons or not?
- variableName or variable_name?
- function () {} or () => {}?
**- Remove comments and make sure you have a good naming and commit before pushing anything to the branch**

**Review by vicnguyenqt — APPROVED** [review]
LGTM!

**Review by thinhnguyen5 — APPROVED** [review]

**Pull request merged**